node/src/components/consensus/era_supervisor.rs

```diff
@@ -522,6 +524,11 @@ where
             let validators;
             let start_height;
             let era_start_time;
+            let seed;
+
+            let booking_block_hash = booking_blocks
+                .get(&era_id)
+                .expect("should have booking block");
```

Comment:
(This looks surprising… but we can `expect` that because the map _does_ have entries for all eras, and they're just `default()` for the ones without booking blocks, right?)

Reply:
Yes. At least, that's the plan.